### DISCUSSION TITLE
勇敢是最宝贵的品质，致敬所有勇敢的人~

### DISCUSSION THREAD

**Discussion by xiaer93** [WeNeedHome/SummaryOfLoanSuspension#482] [General]
勇气，捅破天花板好难，坚守法制不断提高勇气的地板~

**Comment by Rigidbody911** [comment]
能问出这种问题说明你的心里还是指望青天大老爷帮你做主，事实就是拆钠只有绝对的权力导致人治没有法治

**Comment by 0xhelium** [comment]
你说言论自由我都觉得有点搞笑.
一个不存在的东西, 谈什么边界线. 唱赞歌就好了, 现在的一片天, 是晴朗的一片天!

**Reply by TALK1949** [reply]
确实，都没有的东西谈什么边界，我建议先了解一下什么叫做言论自由，再来讨论这个问题。
https://zh.wikipedia.org/wiki/%E8%A8%80%E8%AB%96%E8%87%AA%E7%94%B1

言论自由，一种基本人权，指公民可以按照个人意愿表达意见和想法的法定政治[权利]，这些意见表达不用受任何人“事前”的审查及限制，也无需担心受到政府或他人报复。

**Reply by lettermessenger** [reply]
红日升在东方，其大道满霞光

**Comment by liuwenwu520** [comment]
每个地方都有一定的言论不自由，但是现在的中国言论太不自由 了，很多反对的声音都不能存在，以后压迫多了后肯定会有更多的反抗，到时候可能会言论自由一点

**Reply by MYWmiss** [reply]
压迫多了就习惯啦~

**Reply by lance111l** [reply]
特朗普的账号能封 你的账号当然能封

**Reply by Cegazhos** [reply]
> 特朗普的账号能封 你的账号当然能封

那是推特ban的。特朗普还能发其他平台甚至自建一个平台去说话。

**Comment by rechardlc** [comment]
言论自由是相对的？任何国家地区都没有绝对的自由？只是西朝鲜过度限制了~

**Comment by shawn-yee** [comment]
言论自由的空间在哪里?

**Comment by xiaer93** [comment]


**Reply by clleo888** [reply]
言论自由的边界线非常清楚: 
一切赞美党的可以随便讲; 
一切对党不利的, 有可能造成不利的, 包括沉默, 都不被允许

**Reply by zongjiangU** [reply]
中国言论边界？
连TM法律都没边界了，民法典里多少的所谓 “其他情形”、“特殊情形”  其他和特殊依据不就是靠主观意识吗，人治替代法制

**Reply by Cegazhos** [reply]
> 言论自由的边界线非常清楚: 一切赞美党的可以随便讲; 一切对党不利的, 有可能造成不利的, 包括沉默, 都不被允许

赞美也必须经过审批哦

**Comment by blisee** [comment]
边界很清楚
顺我者昌逆我者亡

**Comment by elevenjiang1** [comment]
非常好奇，最终被Ban的理由是什么呢？法治社会至少也要给个道理吧？
宣传虚假消息还是危害国家安全什么的？
（认真发问，感觉国内有太多被Ban的消息了，但是好像都没有说为什么被Ban）

**Reply by rechardlc** [reply]
不ban，90%的人知道，ban了~10%的人知道~留下美好信念不好嘛？

**Reply by C-Joey** [reply]
为什么要有理由 没有理由寻衅滋事能不能用呢 破坏金融稳定 勾结境外势力 这些不是都能用的

**Reply by pengyidong** [reply]
不符合统治阶级伟光正的形象

**Reply by ghost** [reply]
你支还是法治社会？

**Reply by WE-color** [reply]
中国是人治社会，就是因为不能做到依法治国才天天喊“法治”，懂吗！你看漂亮国天天喊依法治国不？政府说你有问题就是，不需要理由；

**Comment by ryzencool** [comment]
法无禁止即自由 。

**Reply by zhaomingtao-fighting** [reply]
什么法？领导的想法吧

**Reply by pengyidong** [reply]
你说法治我都觉得好笑.jpg

**Reply by xijinping0** [reply]
> 什么法？领导的想法吧

三个基本法：领导的看法、领导的想法、领导的说法

**Comment by John-zhan** [comment]
有边界的东西还能叫自由吗

**Comment by redapple2021** [comment]
我认为只要不是赞歌，歌颂党歌颂伟大领袖，都有可能是红线

**Comment by Novite5** [comment]
言论管得越严格，规则制定越模糊，宣传系统的权力才能越大，政治资源和油水才能越多。
网上某些“反贼”也不排除是宣传系统在养寇自重。

**Comment by ghost** [comment]
言论自由一旦有边界，你就要把定义边界的权力交到政府、跨国公司手里。

**Comment by yanwuguangjun** [comment]
言论自由又怎么样   即使是美国 普通人又能看到多少真相

**Reply by TheGreateJohny** [reply]
笑话了，非常经典的whataboutism了。

**Reply by wolfiesonfire** [reply]
乌鸦笑猪黑

**Comment by volans19** [comment]
认真回答你的问题

边界线就是中国和朝鲜的国境线，只是全世界在边界里，中国在界外

**Comment by zhaipro** [comment]
一个中国等对外的原则性话题不得乱说，对内的问题就要看动机了。

**Comment by xiaer93** [comment]


**Reply by Siryedu** [reply]
这个应该没人反驳吧，毕竟都上github了

**Reply by brianzrk** [reply]
![image](https://user-images.githubusercontent.com/48397581/181176378-d4c5b94e-9c3d-4224-9ee2-6cd57f589466.png)

**Comment by Tentiauhong** [comment]
相信黨相信政府就是中國人那小小的自由邊際線了，我看這github項目也快爛尾。你們保重了！

**Comment by dofindme** [comment]
言论自由（英语：Freedom of speech），一种基本人权，指公民可以按照个人意愿表达意见和想法的法定政治[权利](https://zh.wikipedia.org/wiki/%E6%AC%8A%E5%88%A9)，**这些意见表达不用受任何人“事前”的审查及限制**，也无需担心受到政府或他人报复。有时也被称为意涵更广泛的表达自由

**Comment by xujiyou** [comment]
大兴文字狱

**Comment by yellowcaicai** [comment]
边界就是当权者的利益

**Comment by yellowcaicai** [comment]
国人太听话了，只能由着他们乱来，，你放西方国家试试

**Comment by JayChou55** [comment]
这个自由 主要针对的事 zf匹配监督的言论自由

**Comment by winnoryder** [comment]
言论自由的边界就是道德的约束范围，道德的约束范围在几乎所有的正教中都有一个轮廓的概述。

**Comment by Frkie** [comment]
不太明白为啥不能讨论政治，这种思想就是错误的。

**Comment by TurnOffNOD** [comment]
言论自由的边界应该是符合事实，而且是符合事实的全貌。

**Comment by baifachuan** [comment]
`
《人民日报》5月5日的一篇社论说：【有人认为民主就是可以为所欲为，但我们主张的是在集体指导下的民主】
`

出自《邓小平时代》第8章，作者傅高义。

**Comment by naifenmizuha** [comment]
川主席被推特封号想必是**极其**_言论自由_的

**Comment by Cegazhos** [comment]
不涉及到政治的还是能说的。所以我支持踩油门

**Comment by hclgdb** [comment]
允许批评！

**Reply by xiaer93** [reply]
若批评不自由，则赞美无意义。

**Comment by b4o** [comment]
这个世界哪里 有绝对了言论自由？有吗？哪个国家？原始社会才有的东西

**Reply by ITCNZ** [reply]
所以只能允许赞美，一丁点质疑都不可以？

**Comment by xiaer93** [comment]


**Reply by xijinping0** [reply]
> 知道国家主席是谁吗？

知道啊。不仅知道现在的国家主席是谁，还知道下一任主席是谁，就是我习近平！充分展现我国低民主优势。

**Comment by lance111l** [comment]
言论自由的边界线在媒体的控制  媒体是无冕之王 
特朗普总统的账号都能被封     你的自由?
美国权力中心被多人强行进入  幕后没人负责   你的自由?
